content/starter-configs/tutorials/testery.md

```diff
@@ -0,0 +1,169 @@
+      # from the base preview.
+      build:
+        # Check for upgrades to the Testery command line interface
+        - pip3 install testery --upgrade
+        # Set up Testery to run tests against the Tugboat environment, using Tugboat environment variables
+        - testery update-environment --create-if-not-exists --token "$TESTERY_TOKEN" --key "${TUGBOAT_PREVIEW}" --name
+          "${TUGBOAT_PREVIEW}" --variable "TUGBOAT_DEFAULT_SERVICE_URL=${TUGBOAT_DEFAULT_SERVICE_URL}"
+
+        # Start a test run.
+        - testery create-test-run --token "$TESTERY_TOKEN" --git-ref "$TUGBOAT_PREVIEW_SHA" --project "testery"
+          --environment "${TUGBOAT_PREVIEW}"
+
+  # What to call the service hosting MySQL. This name also acts as the
+  # hostname to access the service by from the php service.
+  mysql:
+    # Use the latest available 5.x version of MySQL
+    image: tugboatqa/mysql:5
+
+    # A set of commands to run while building this service
+    commands:
+      # Commands that import files, databases,  or other assets. When an
+      # existing preview is refreshed, the build workflow starts here,
+      # skipping the init step, because the results of that step will
+      # already be present.
+      update:
+        # Copy a database dump from an external server. The public
+        # SSH key found in the Tugboat Repository configuration must be
+        # copied to the external server in order to use scp.
+        - curl -L "https://www.dropbox.com/s/sabj5vq711bhst2/demo-wordpress-database.sql.gz?dl=0" > /tmp/database.sql.gz
+        - zcat /tmp/database.sql.gz | mysql tugboat
+        - rm /tmp/database.sql.gz
+```
+
+With these steps in place, and depending on configurations you set up in both services, you'll then be able to see the
+results of your Tugboat builds _and_ your Testery test runs on your git pull requests:
+
+![Screenshot: Testery test results and Tugboat deploy on a pull request in GitHub](/_images/testery-and-tugboat-on-pr.png)
+
+Want to know more about something mentioned in the comments of this config file? Check out these topics:
+
+- [Name your Service](/setting-up-services/how-to-set-up-services/name-your-service/)
+- [Specify a Service image](/setting-up-services/how-to-set-up-services/specify-a-service-image/)
+- [Leverage Service commands](/setting-up-services/how-to-set-up-services/leverage-service-commands/)
+- [Define a default Service](/setting-up-services/how-to-set-up-services/define-a-default-service/)
+- [Set the document root path](/setting-up-services/how-to-set-up-services/set-the-document-root-path/)
+- [Set up remote SSH access](/setting-up-tugboat/select-repo-settings/#set-up-remote-ssh-access)
+- [Preview build process phases (`init`, `update`, `build`)](/building-a-preview/preview-deep-dive/how-previews-work/#the-build-process-explained)
+- [How Base Previews work](/building-a-preview/preview-deep-dive/how-previews-work/#how-base-previews-work)
+
+## Start Building Previews!
```

Comment:
Maybe we should say "Start testing your Previews!" or something? Or just remove this paragraph?